{"uid":"ce6b0a383cfd6aae","name":"Test create user","fullName":"tests.test_users.TestUsersAPI#test_create_user","historyId":"7906fed064abe2a7a41f166381015ab4","time":{"start":1733115639600,"stop":1733115639600,"duration":0},"description":"Verify user creation and data validation","descriptionHtml":"<p>Verify user creation and data validation</p>\n","status":"failed","statusMessage":"AssertionError: User data validation failed: 1 validation error for UserResponse\nnickname\n  String should have at least 2 characters [type=string_too_short, input_value='', input_type=str]\n    For further information visit https://errors.pydantic.dev/2.10/v/string_too_short","statusTrace":"self = <tests.test_users.TestUsersAPI object at 0x7f99cbefea80>\nnew_user = {'avatar_url': '', 'email': 'longangela@example.net', 'name': 'Jennifer', 'nickname': '', ...}\n\n    @allure.title(\"Test create user\")\n    @allure.description(\"Verify user creation and data validation\")\n    @allure.severity(allure.severity_level.CRITICAL)\n    def test_create_user(self, new_user: dict) -> None:\n        \"\"\"\n        Test creating a new user.\n    \n        :param new_user: Fixture with a newly created user\n        \"\"\"\n        with allure.step(\"Validating newly created user data\"):\n            user = new_user\n    \n            try:\n>               UserResponse.model_validate(user)\nE               pydantic_core._pydantic_core.ValidationError: 1 validation error for UserResponse\nE               nickname\nE                 String should have at least 2 characters [type=string_too_short, input_value='', input_type=str]\nE                   For further information visit https://errors.pydantic.dev/2.10/v/string_too_short\n\ntests/test_users.py:112: ValidationError\n\nThe above exception was the direct cause of the following exception:\n\nself = <tests.test_users.TestUsersAPI object at 0x7f99cbefea80>\nnew_user = {'avatar_url': '', 'email': 'longangela@example.net', 'name': 'Jennifer', 'nickname': '', ...}\n\n    @allure.title(\"Test create user\")\n    @allure.description(\"Verify user creation and data validation\")\n    @allure.severity(allure.severity_level.CRITICAL)\n    def test_create_user(self, new_user: dict) -> None:\n        \"\"\"\n        Test creating a new user.\n    \n        :param new_user: Fixture with a newly created user\n        \"\"\"\n        with allure.step(\"Validating newly created user data\"):\n            user = new_user\n    \n            try:\n                UserResponse.model_validate(user)\n            except ValidationError as e:\n                msg = f\"User data validation failed: {e}\"\n>               raise AssertionError(msg) from e\nE               AssertionError: User data validation failed: 1 validation error for UserResponse\nE               nickname\nE                 String should have at least 2 characters [type=string_too_short, input_value='', input_type=str]\nE                   For further information visit https://errors.pydantic.dev/2.10/v/string_too_short\n\ntests/test_users.py:115: AssertionError","flaky":true,"newFailed":true,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"name":"_verify_url","time":{"start":1733115630864,"stop":1733115630864,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"_session_faker","time":{"start":1733115630864,"stop":1733115630865,"duration":1},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"base_url","time":{"start":1733115630864,"stop":1733115630864,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"delete_output_dir","time":{"start":1733115630864,"stop":1733115630864,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"pytestconfig","time":{"start":1733115630864,"stop":1733115630864,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"playwright","time":{"start":1733115630866,"stop":1733115631104,"duration":238},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"api_request_context","time":{"start":1733115631104,"stop":1733115631112,"duration":8},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"user_api_client","time":{"start":1733115639106,"stop":1733115639106,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"new_user","time":{"start":1733115639107,"stop":1733115639599,"duration":492},"status":"passed","steps":[],"attachments":[{"uid":"c938fa5e42eba950","name":"API Response","source":"c938fa5e42eba950.txt","type":"text/plain","size":159},{"uid":"d9f9a0534a24f3a8","name":"Status Code","source":"d9f9a0534a24f3a8.txt","type":"text/plain","size":3},{"uid":"9059b80937e962ca","name":"Headers","source":"9059b80937e962ca.txt","type":"text/plain","size":648},{"uid":"e6544037d72ecef6","name":"URL","source":"e6544037d72ecef6.txt","type":"text/plain","size":45}],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":4,"hasContent":true,"attachmentStep":false}],"testStage":{"description":"Verify user creation and data validation","status":"failed","statusMessage":"AssertionError: User data validation failed: 1 validation error for UserResponse\nnickname\n  String should have at least 2 characters [type=string_too_short, input_value='', input_type=str]\n    For further information visit https://errors.pydantic.dev/2.10/v/string_too_short","statusTrace":"self = <tests.test_users.TestUsersAPI object at 0x7f99cbefea80>\nnew_user = {'avatar_url': '', 'email': 'longangela@example.net', 'name': 'Jennifer', 'nickname': '', ...}\n\n    @allure.title(\"Test create user\")\n    @allure.description(\"Verify user creation and data validation\")\n    @allure.severity(allure.severity_level.CRITICAL)\n    def test_create_user(self, new_user: dict) -> None:\n        \"\"\"\n        Test creating a new user.\n    \n        :param new_user: Fixture with a newly created user\n        \"\"\"\n        with allure.step(\"Validating newly created user data\"):\n            user = new_user\n    \n            try:\n>               UserResponse.model_validate(user)\nE               pydantic_core._pydantic_core.ValidationError: 1 validation error for UserResponse\nE               nickname\nE                 String should have at least 2 characters [type=string_too_short, input_value='', input_type=str]\nE                   For further information visit https://errors.pydantic.dev/2.10/v/string_too_short\n\ntests/test_users.py:112: ValidationError\n\nThe above exception was the direct cause of the following exception:\n\nself = <tests.test_users.TestUsersAPI object at 0x7f99cbefea80>\nnew_user = {'avatar_url': '', 'email': 'longangela@example.net', 'name': 'Jennifer', 'nickname': '', ...}\n\n    @allure.title(\"Test create user\")\n    @allure.description(\"Verify user creation and data validation\")\n    @allure.severity(allure.severity_level.CRITICAL)\n    def test_create_user(self, new_user: dict) -> None:\n        \"\"\"\n        Test creating a new user.\n    \n        :param new_user: Fixture with a newly created user\n        \"\"\"\n        with allure.step(\"Validating newly created user data\"):\n            user = new_user\n    \n            try:\n                UserResponse.model_validate(user)\n            except ValidationError as e:\n                msg = f\"User data validation failed: {e}\"\n>               raise AssertionError(msg) from e\nE               AssertionError: User data validation failed: 1 validation error for UserResponse\nE               nickname\nE                 String should have at least 2 characters [type=string_too_short, input_value='', input_type=str]\nE                   For further information visit https://errors.pydantic.dev/2.10/v/string_too_short\n\ntests/test_users.py:115: AssertionError","steps":[{"name":"Validating newly created user data","time":{"start":1733115639600,"stop":1733115639600,"duration":0},"status":"failed","statusMessage":"AssertionError: User data validation failed: 1 validation error for UserResponse\nnickname\n  String should have at least 2 characters [type=string_too_short, input_value='', input_type=str]\n    For further information visit https://errors.pydantic.dev/2.10/v/string_too_short\n","statusTrace":"  File \"/home/runner/work/qa-hackathon-raptors/qa-hackathon-raptors/tests/test_users.py\", line 115, in test_create_user\n    raise AssertionError(msg) from e\n","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":true,"stepsCount":0,"attachmentsCount":0,"hasContent":true,"attachmentStep":false}],"attachments":[],"parameters":[],"shouldDisplayMessage":true,"stepsCount":1,"attachmentsCount":0,"hasContent":true,"attachmentStep":false},"afterStages":[{"name":"new_user::0","time":{"start":1733115639605,"stop":1733115640381,"duration":776},"status":"passed","steps":[],"attachments":[{"uid":"3fa06a2e7823f75b","name":"API Response","source":"3fa06a2e7823f75b.txt","type":"text/plain","size":14},{"uid":"5366805a31dfaf64","name":"Status Code","source":"5366805a31dfaf64.txt","type":"text/plain","size":3},{"uid":"733b64601480ca4a","name":"Headers","source":"733b64601480ca4a.txt","type":"text/plain","size":579},{"uid":"9e461d3cfd283c92","name":"URL","source":"9e461d3cfd283c92.txt","type":"text/plain","size":82}],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":4,"hasContent":true,"attachmentStep":false},{"name":"api_request_context::0","time":{"start":1733115650121,"stop":1733115650124,"duration":3},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"playwright::0","time":{"start":1733115650124,"stop":1733115650130,"duration":6},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"labels":[{"name":"severity","value":"critical"},{"name":"feature","value":"User API Operations"},{"name":"epic","value":"User Management"},{"name":"parentSuite","value":"tests"},{"name":"suite","value":"test_users"},{"name":"subSuite","value":"TestUsersAPI"},{"name":"host","value":"fv-az1437-820"},{"name":"thread","value":"5520-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.test_users"},{"name":"resultFormat","value":"allure2"}],"parameters":[{"name":"new_user","value":"'api-3'"}],"links":[],"hidden":false,"retry":false,"extra":{"severity":"critical","retries":[],"categories":[{"name":"Product defects","matchedStatuses":[],"flaky":false}],"history":{"statistic":{"failed":4,"broken":4,"skipped":0,"passed":5,"unknown":0,"total":13},"items":[{"uid":"62384b24ed0d76db","reportUrl":"https://ZhikharevAl.github.io/qa-hackathon-raptors/7//#testresult/62384b24ed0d76db","status":"passed","time":{"start":1733115453794,"stop":1733115453794,"duration":0}},{"uid":"e8f6cdfc55d940dc","reportUrl":"https://ZhikharevAl.github.io/qa-hackathon-raptors/6//#testresult/e8f6cdfc55d940dc","status":"broken","statusDetails":"ValueError: Request failed with status 422","time":{"start":1733114762971,"stop":1733114762971,"duration":0}},{"uid":"9be272cc3fefa366","reportUrl":"https://ZhikharevAl.github.io/qa-hackathon-raptors/5//#testresult/9be272cc3fefa366","status":"broken","statusDetails":"ValueError: Request failed with status 422","time":{"start":1733114594938,"stop":1733114594938,"duration":0}},{"uid":"47e459a8fa537608","reportUrl":"https://ZhikharevAl.github.io/qa-hackathon-raptors/6//#testresult/47e459a8fa537608","status":"broken","statusDetails":"ValueError: Request failed with status 422","time":{"start":1733114575479,"stop":1733114575479,"duration":0}},{"uid":"7398a673c8e743a0","reportUrl":"https://ZhikharevAl.github.io/qa-hackathon-raptors/5//#testresult/7398a673c8e743a0","status":"failed","statusDetails":"AssertionError: User data validation failed: 1 validation error for UserResponse\nnickname\n  String should have at least 2 characters [type=string_too_short, input_value='', input_type=str]\n    For further information visit https://errors.pydantic.dev/2.10/v/string_too_short","time":{"start":1733100687594,"stop":1733100687595,"duration":1}},{"uid":"9c7750861daa7b12","reportUrl":"https://ZhikharevAl.github.io/qa-hackathon-raptors/4//#testresult/9c7750861daa7b12","status":"failed","statusDetails":"AssertionError: User data validation failed: 1 validation error for UserResponse\nnickname\n  String should have at least 2 characters [type=string_too_short, input_value='', input_type=str]\n    For further information visit https://errors.pydantic.dev/2.10/v/string_too_short","time":{"start":1733100080644,"stop":1733100080645,"duration":1}},{"uid":"2f44f00b0c3ee5b6","reportUrl":"https://ZhikharevAl.github.io/qa-hackathon-raptors/4//#testresult/2f44f00b0c3ee5b6","status":"passed","time":{"start":1733094768563,"stop":1733094768563,"duration":0}},{"uid":"649b68d0afced2ec","reportUrl":"https://ZhikharevAl.github.io/qa-hackathon-raptors/3//#testresult/649b68d0afced2ec","status":"passed","time":{"start":1733094542123,"stop":1733094542123,"duration":0}},{"uid":"514f5005efca0de4","reportUrl":"https://ZhikharevAl.github.io/qa-hackathon-raptors/2//#testresult/514f5005efca0de4","status":"passed","time":{"start":1733087333300,"stop":1733087333300,"duration":0}},{"uid":"609aab2feeb85590","reportUrl":"https://ZhikharevAl.github.io/qa-hackathon-raptors/3//#testresult/609aab2feeb85590","status":"failed","statusDetails":"AssertionError: User data validation failed: 1 validation error for UserResponse\nnickname\n  String should have at least 2 characters [type=string_too_short, input_value='', input_type=str]\n    For further information visit https://errors.pydantic.dev/2.10/v/string_too_short","time":{"start":1733086962888,"stop":1733086962889,"duration":1}},{"uid":"9ebf44839eca9f9c","reportUrl":"https://ZhikharevAl.github.io/qa-hackathon-raptors/2//#testresult/9ebf44839eca9f9c","status":"broken","statusDetails":"ValueError: Request failed with status 404","time":{"start":1733086643809,"stop":1733086643809,"duration":0}},{"uid":"da5b9c0ad993a9bb","reportUrl":"https://ZhikharevAl.github.io/qa-hackathon-raptors/1//#testresult/da5b9c0ad993a9bb","status":"passed","time":{"start":1733085920247,"stop":1733085920247,"duration":0}}]},"tags":[]},"source":"ce6b0a383cfd6aae.json","parameterValues":["'api-3'"]}